### ISSUE TITLE
acute repetitive seizures

### ISSUE THREAD

**Issue by OOPD-LJF** [monarch-initiative/mondo#5496]
**Preferred term label: acute repetitive seizures**

**Synonyms: cluster seizures, serial seizures, crescendo seizures, sequential seizures, seizure flurries, recurrent seizures, or cyclical seizures**

**Definition: a pattern of increased seizure frequency or severity which is recognizable to the person/caregiver and which is not the patient's usual seizure pattern. One proposed clinical definition of acute repetitive seizures is three or more seizures within 24 hours for patients whose habitual seizure frequency is fewer than three seizures per day. (PMID: 30921018)**

**Parent term: nervous system disorder**

**Children terms: N/A**

**Your nano-attribution (ORCID): 0000-0001-7151-1615**

**Comment by sabrinatoro** [comment]
We should look into this issue with https://github.com/monarch-initiative/mondo/issues/5957

**Comment by nicolevasilevsky** [comment]
@OOPD-LJF this is phenotype in HPO: https://hpo.jax.org/app/browse/term/HP:0033349

Could you use this term or do you need a separate disease term to be added to Mondo? If so, should this be a child of epilepsy?

**Comment by OOPD-LJF** [comment]
Thanks Nicole - it does look like this is probably a phenotype - I don't think it needs to be added to Mondo.

**Comment by nicolevasilevsky** [comment]
Sounds good, thanks @OOPD-LJF!
